src/config/config.json

```diff
@@ -0,0 +1,25 @@
+{
+  "db": {
+    "development": {
+      "database": "express_nuxt_dev_db",
+      "host": "localhost",
```

Comment:
Since this is dependent on environment, maybe it will be more appropriate to put in .env variables ?